cli/internal/state/state.go

```diff
@@ -164,3 +165,27 @@ func (s *State) Merge(other *State) (*State, error) {
 	}
 	return s, nil
 }
+
+// hexBytes is a byte slice that is marshalled to and from a hex string.
+type hexBytes []byte
+
+// UnmarshalYAML implements the yaml.Unmarshaler interface.
+func (h *hexBytes) UnmarshalYAML(unmarshal func(any) error) error {
+	var hexString string
+	if err := unmarshal(&hexString); err != nil {
+		return err
+	}
```

Comment:
```suggestion
// UnmarshalYAML implements the yaml.Unmarshaler interface.
func (h *hexBytes) UnmarshalYAML(unmarshal func(any) error) error {
	var hexString string
	if err := unmarshal(&hexString); err != nil {
		var hexBytes []byte
		if err := unmarshal(&hexBytes); err != nil {
			return err
		}
		hexString = hex.EncodeToString(hexBytes)
	}
```
Since we already released a version of Constellation that supported a statefile, we need compatibility code here to make sure we can unmarshal from the yaml array